### PR TITLE
add latest draft for validation, use comment

### PR DESCRIPTION
**Pull request by foxriver76**

### CODE REVIEW

File: schemas/io-package.json [hunk 1]
```diff
@@ -65,13 +80,22 @@
                     "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
                     "type": "boolean"
                 },
+                "readme": {
+                    "description": "Path to the ReadMe file",
```
Comment (AlCalzone) [inline schemas/io-package.json:84]: relative local file path or absolute URL?

File: schemas/io-package.json [hunk 2]
```diff
@@ -91,15 +115,40 @@
                 "license": {
                     "description": "License of the software",
                     "type": "string",
-                    "pattern": "(?i)afl-3.0|apache-2.0|artistic-2.0|bsl-1.0|bsd-2-clause|bsd-3-clause|bsd-3-clause-clear|cc|cc0-1.0|cc-by-4.0|cc-by-sa-4.0|wtfpl|ecl-2.0|epl-1.0|epl-2.0|eupl-1.1|agpl-3.0|gpl|gpl-2.0|gpl-3.0|lgpl|lgpl-2.1|lgpl-3.0|isc|lppl-1.3c|ms-pl|mit|mpl-2.0|osl-3.0|postgresql|ofl-1.1|ncsa|unlicense|zlib"
+                    "pattern": "afl-3.0|AFL-3.0|apache-2.0|APACHE-2.0|artistic-2.0|ARTISTIC-2.0|bsl-1.0|BSL-1.0|bsd-2-clause|BSD-2-CLAUSE|bsd-3-clause|BSD-3-CLAUSE|bsd-3-clause-clear|BSD-3-CLAUSE-CLEAR|cc|CC|cc0-1.0|CC0-1.0|cc-by-4.0|CC-BY-4.0|cc-by-sa-4.0|CC-BY-SA-4.0|wtfpl|WTFPL|ecl-2.0|ECL-2.0|epl-1.0|EPL-1.0|epl-2.0|EPL-2.0|eupl-1.1|EUPL-1.1|agpl-3.0|AGPL-3.0|gpl|GPL|gpl-2.0|GPL-2.0|gpl-3.0|GPL-3.0|lgpl|LGPL|lgpl-2.1|LGPL-2.1|lgpl-3.0|LGPL-3.0|isc|ISC|lppl-1.3c|LPPL-1.3C|ms-pl|MS-PL|mit|MIT|mpl-2.0|MPL-2.0|osl-3.0|OSL-3.0|postgresql|POSTGRESQL|ofl-1.1|OFL-1.1|ncsa|NCSA|unlicense|UNLICENSE|zlib|ZLIB"
```
Comment (AlCalzone) [inline schemas/io-package.json:118]: Wow, there is really no case-insensitive flag 😕

Comment (foxriver76) [reply schemas/io-package.json:118]: Unfortunately yes, it worked at least in my IDE but it's not officially supported and will crash tests from schemastore

File: schemas/io-package.json (hunk 1 above)
Comment (foxriver76) [reply schemas/io-package.json:84]: url works for sure, path not sure, will better change it

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**